views/__init__.py

```diff
@@ -215,10 +215,6 @@ def change_password():
         print 'LOGIN SUCCESS, CHANGING PASSWORD'
         argon_password = argon2.hash(new_password_1)
         db_users = get_db(app.config['DB_NAME_USERS'])
-        #db_users.users.update_one({'user':username},{'$set':{'password':hash}})
-        q={'query':'MATCH (u:User {user: $user}) SET u.password=$password','parameters':{'user':username,'password':password}}
```

Comment:
@pontikos, just making sure this is fine?

Is there any reason for the clear text password to be stored into the neo4j db?